python/README.md

```diff
@@ -2,68 +2,77 @@
+- Intel<sup>®</sup> CPUs
+- Intel<sup>®</sup> integrated GPUs
+- Intel<sup>®</sup> Movidius™ Vision Processing Units - referred to as VPU
+- Intel<sup>®</sup> Vision Accelerator Design with 8 Intel Movidius™ MyriadX VPUs - referred to as VAD-M or HDDL
+
+[Note: For maximum performance, efficiency, tooling customization, and hardware control, we recommend the developers to adopt native OpenVINO™ APIs and its runtime.]
 
 ## Installation
 
 ### Requirements
 
-- Python 3.6, 3.7, or 3.8
-- TensorFlow 2.5.1
-
-### Use **OpenVINO™ integration with TensorFlow** alongside PyPi TensorFlow
+- Ubuntu 18.04, 20.04 or macOS 11.2.3
+- Python* 3.6, 3.7, 3.8 or 3.9
+- TensorFlow* v2.5.1
 
-This **OpenVINO™ integration with TensorFlow** package comes with pre-built libraries of OpenVINO™ version 2021.4.1. The users do not have to install OpenVINO™ separately. This package supports Intel<sup>®</sup> CPUs, Intel<sup>®</sup> integrated GPUs and Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs).
+This **OpenVINO™ integration with TensorFlow** package comes with pre-built libraries of OpenVINO™ version 2021.4.1 meaning you do not have to install OpenVINO™ separately.
+This package supports:
+- Intel<sup>®</sup> CPUs
+- Intel<sup>®</sup> integrated GPUs
+- Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs)
 
         pip3 install pip==21.0.1
         pip3 install tensorflow==2.5.1
         pip3 install -U openvino-tensorflow
 
-To use **OpenVINO™ integration with TensorFlow** with pre-installed OpenVINO™ binaries, please visit this page for detailed instructions: ([**OpenVINO™ integration with TensorFlow** - README](docs/BUILD.md))
+To leverage Intel® Vision Accelerator Design with Movidius™ (VAD-M) for inference, please refer to: [**OpenVINO™ integration with TensorFlow** alongside the Intel® Distribution of OpenVINO™ Toolkit](https://github.com/openvinotoolkit/openvino_tensorflow/blob/master/docs/INSTALL.md#12-install-openvino-integration-with-tensorflow-alongside-the-intel-distribution-of-openvino-toolkit).
+
+For more details on installation please refer to [INSTALL.md](https://github.com/openvinotoolkit/openvino_tensorflow/blob/master/docs/INSTALL.md), and for build from source options please refer to [BUILD.md](https://github.com/openvinotoolkit/openvino_tensorflow/blob/master/docs/BUILD.md)
```

Comment:
IINSTALL.md link seems to be not working

Reply:
As the link points to master, and master doesn't have this file yet, it fails.